src/app/loadout-builder/LoadoutBuilder.m.scss

```diff
@@ -9,11 +9,15 @@
 }
 
 .menuContent {
-  box-sizing: border-box;
-
```

Comment:
This change breaks mobile dimensions

https://github.com/DestinyItemManager/DIM/assets/7344652/35b18774-2e7d-4df9-925d-5d60cdbf5ff3